### DISCUSSION TITLE
为什么卡巴斯基检测到软件有病毒？

### DISCUSSION THREAD

**Discussion by Bpc-CPU** [natfrp/wiki#414] [使用问题] [answered]
从
https://www.natfrp.com/tunnel/download
下载的软件，难道是官网被黑了吗？连带木马的软件的md5和官网上的都对的上。附图。
![%7BDF492F2B-AD13-48a3-8173-4639C669DBA8%7D](https://github.com/natfrp/wiki/assets/62590333/7dbd0b65-049f-4da0-9f49-9c2830cc4f7b)

[a comment by an automated account is not shown]

**Reply by Bpc-CPU** [reply]
可是我打开sakurafrp，它会响起一段背景音乐（木马程序特有的那种），而且它还修改权限，我都打不开浏览器和任务管理器。

**Reply by fakeboboliu** [reply]
> 可是我打开sakurafrp，它会响起一段背景音乐（木马程序特有的那种），而且它还修改权限，我都打不开浏览器和任务管理器。

我们无法确认您的系统情况，我们的卡巴斯基 21.3 和 21.15 用户并不会报毒也不会出现这样的问题，建议您检查自己的系统和文件。

常见的排除方案包括但不限于：
 - 启动全盘查杀
 - 将启动器安装包移动到另一个目录运行

**Reply by Bpc-CPU** [reply]
好的，我现在用命令行版本。我估计是别人利用有ui的frpc来植入木马了，谢谢。
